### PR TITLE
Feat/homepage

### PR DESCRIPTION
**Pull request by Debarshi95**
PR to add homepage with related components

Preview: https://626581f91aae254e7d38b4df--debarshib-wetube.netlify.app

### CODE REVIEW

File: src/components/Card/Card.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.Card__root {
+  background-color: green;
```
Comment (saurabhchirde) [inline src/components/Card/Card.css:2]: Hard coding color can be avoided.

File: src/components/Text/Text.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  }
+
+  .Text--xxl {
+    font-size: 1.9rem;
+  }
+  .Text--xxxl {
+    font-size: 2rem;
+  }
+  .Text--xxlg {
+    font-size: 2.3rem;
```
Comment (saurabhchirde) [inline src/components/Text/Text.css:109]: This is something new for me, couldn't think that we can also do this 😅

File: src/pages/App.js [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
-  return <div className="App">App</div>;
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route
+          path="/"
+          index
+          element={
+            <Suspense fallback={<h1>Loading</h1>}>
```
Comment (saurabhchirde) [inline src/pages/App.js:16]: "Suspense" This is something new I found, would have to read about this.

**Review by saurabhchirde — APPROVED** [review]
Couldn't find any issues in your code. In fact, I learned some new things from your code. 
Good to merge.

**Review by sanket4120 — APPROVED** [review]
Good to merge

**Pull request merged**